relay/kusama/src/lib.rs

```diff
@@ -181,7 +181,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: alloc::borrow::Cow::Borrowed("kusama"),
 	impl_name: alloc::borrow::Cow::Borrowed("parity-kusama"),
 	authoring_version: 2,
-	spec_version: 1_004_003,
+	spec_version: 1_005_000,
```

Comment:
is there a reason why we're not also bumping ParachainHost runtime api to 13?

The relay chain node assumes it's set to 3 if it's not present so elastic scaling should still work. But I'm curious as to why we didn't implement v13

Reply:
ah nevermind, you pointed out in private that we didn't yet upgrade to stable2503